### PR TITLE
Removing GitPython from tests

### PR DESCRIPTION
**Pull request by Radinyn**

### CODE REVIEW

File: protostar/git/git_repository.py [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+    InvalidGitRepositoryException,
+    wrap_git_exception,
+)
+
+
+@dataclass
+class Submodule:
+    name: str
+    url: str
+    version: Optional[str] = None
```
Comment (kasperski95) [inline protostar/git/git_repository.py:17]: Nit: `Submodule::version` doesn't make sense. Consider `PackageInfo.from_submodule` to allow these two dataclasses to vary independently.

**Review by kasperski95 — DISMISSED** [review]

File: protostar/git/git_repository.py (hunk 1 above)
Comment (Radinyn) [reply protostar/git/git_repository.py:17]: I don't think it's particularly important at the moment. I might do it in a separate PR, perhaps related to package version enforcing.

**Comment by kasperski95** [comment]
10:59:09 [ERROR] Installation failed
10:59:09 [ERROR] Error while executing Git command:
Command '['git', 'commit', '-m', 'add cairo_contracts']' returned non-zero exit status 1.
10:59:09 [INFO] Execution time: 0.96 s

**Review by kasperski95 — APPROVED** [review]
This PR fixes the issue with installing the specific version. However, I've managed to crash protostar by running:
```
protostar install ...
protostar remove ...
protostar install ...
```
That problem can be solved in the different PR.

**Pull request merged**